src/makeTxn.ts

```diff
@@ -206,6 +210,7 @@ export function makeKeyRegistrationTxnWithSuggestedParams(
     type: TransactionType.keyreg,
```

Comment:
you didn't make these changes, but do you know why type and reKeyTo are are labeled but the others are not?

Reply:
`type` is specified because it should always be `keyreg`, and `reKeyTo` is specified because unfortunately it's defined as `reKeyTo` and `rekeyTo` (notice the difference in the k) in different places, so we need to convert it.